script/helperFunctions.sh

```diff
@@ -915,6 +904,9 @@ function saveDiamondFacets() {
   ENVIRONMENT=$2
   USE_MUTABLE_DIAMOND=$3
   FACETS=$4
```

Comment:
to keep variable declarations consistent I woukd suggest changing these to use the same 
local VAR_NAME="$<number>" format as below example: `local ENVIRONMENT="$2"` etc

Reply:
agreed, updated